docs/src/tutorials/pluscal-tutorial.md

```diff
@@ -0,0 +1,241 @@
+# Tutorial on checking Pluscal specifications with Apalache
+
+**Difficulty: Blue trail – Easy**
+
+In this short tutorial, we show how to annotate a [Pluscal specification][] of
+the Bakery algorithm, to check it with Apalache. Further, we check mutual
+exclusion by bounded model checking for bounded executions. Moreover, we
+automatically prove mutual exclusion by induction and bounded model checking
+for unbounded executions.
+
+We only focus on the part
+related to Apalache. If you want to understand the Bakery algorithm and its
+specification, check the comments in the original [Pluscal specification][].
+
+## Related documents
+
+ - [Entry-level Tutorial on the Model Checker][]
+ - [Tutorial on Snowcat][] shows how to write type annotations for Apalache.
+ - [TLA+ Cheatsheet in HTML][] summarizes the common TLA+ constructs. If you
+   prefer a printable version in pdf, check the [Summary of TLA+][].
+
+## Setup
+
+We assume that you have Apalache installed. If not, check the manual page on
+[Apalache installation][]. The minimal required version is 0.22.0.
+
+## Running example: Bakery
+
+We are starting with the [Pluscal specification][] of the Bakery algorithm.
+This specification has been checked with the model checker [TLC][].
+Moreover, [Leslie Lamport][] has proved safety of this algorithm with
+the [TLAPS][].
+
+
+## Step 0: Remove the TLAPS proof
+
+Since we are not interested in the TLAPS proof, we copy [Bakery.tla][] to
+[BakeryWoTlaps.tla][] and modify it as follows:
+
+ - Remove `TLAPS` from the list of extended modules
+
+    ```tla
+    EXTENDS Naturals
+    ```
+
+ - Remove the theorem and its proof:
+
+    ```tla
+    THEOREM Spec => []MutualExclusion
+    ...
+    ```
+
+Alternatively, you can download the module `TLAPS.tla`.
```

Comment:
Provide a link? Maybe this sentence is not needed.

Reply:
I also thought this may not be needed.